Assets/Scripts/Cards/DeckManager.cs

```diff
@@ -149,8 +149,10 @@ public class DeckManager : NetworkSingleton<DeckManager> {
     }
 
     public void DrawInitialHandOfCards() {
-        for (int i = 0; i < GameSettings.initialPlayerHandSize; i++) {
-            DrawCard();
+        //Didn't want to use update since this is only called once
+        for (float i = GameSettings.initialPlayerHandSize; i > 0; i--) {
+            Invoke("DrawCard",(i-(.5f*i)));
+            Debug.Log("invoked at" + (i - (.5f*i)));

```

Comment:
Can you instead pass in a boolean to make this a little cleaner? Something like: `public void DrawCard(bool shouldAnimate = true)`, then you can update the this declaration with `DrawCard(false)`